CMakeLists.txt

```diff
@@ -158,34 +137,32 @@ if(REDNER_CUDA)
         shape.cpp
         sobol_sampler.cpp
         PROPERTIES CUDA_SOURCE_PROPERTY_FORMAT OBJ)
-    set(CUDA_NVCC_FLAGS "${CUDA_NVCC_FLAGS} -std=c++11")
 
     cuda_add_library(redner MODULE ${SRCS})
-    target_link_libraries(redner
-        ${EMBREE_LIBRARY}
-        ${optix_prime_LIBRARY}
-        ${DYNAMIC_LOOKUP})
+    target_link_libraries(redner ${optix_prime_LIBRARY})
 else()
-    # No CUDA
-    if (NOT WIN32)
-        add_library(redner MODULE ${SRCS})
-    
-        # The "-undefined dynamic_lookup" is a hack for systems with
-        # multiple Python installed. If we link a particular Python version
-        # here, and we import it with a different Python version later.
-        # likely a segmentation fault.
-        # The solution for Linux/Mac OS machines, as mentioned in 
-        # https://github.com/pybind/pybind11/blob/master/tools/pybind11Tools.cmake
-        # is to not link against Python library at all and resolve the symbols
-        # at compile time.
-        target_link_libraries(redner
-            ${EMBREE_LIBRARY}
-            ${DYNAMIC_LOOKUP})
-    else()
-        target_link_libraries(redner
-            PRIVATE
-            ${EMBREE_LIBRARY})
-    endif()
+    add_library(redner MODULE ${SRCS})
+endif()
+
+if(APPLE)
+    # The "-undefined dynamic_lookup" is a hack for systems with
+    # multiple Python installed. If we link a particular Python version
+    # here, and we import it with a different Python version later.
+    # likely a segmentation fault.
```

Comment:
This sentence seems to be missing the beginning.

Reply:
It's copied one by one from the original (see https://github.com/BachiLi/redner/pull/103/files#diff-af3b638bc2a3e6c650974192a53c7291L173-L180)

Reply:
Probably a merge error or I was just typing non-sense. Will fix it at some point.